### PR TITLE
Expire for rust

### PR DESCRIPTION
**Pull request by AlexeyVavilin**
Message expiration supported
Async calls

### CODE REVIEW

File: ton_sdk/src/lib.rs [hunk 1]
```diff
@@ -14,37 +14,17 @@
 
 #![recursion_limit="128"] // needed for error_chain
```
Comment (Sumrachek) [inline ton_sdk/src/lib.rs]: Try to delete it

File: ton_sdk/src/lib.rs [hunk 2]
```diff
@@ -72,45 +52,31 @@ pub use local_tvm::executor::TransactionFees;
...
-*/
+
 mod types;
 pub use types::*;
 
 #[cfg(feature = "node_interaction")]
-pub mod queries_helper;
+pub mod node_client;
+#[cfg(feature = "node_interaction")]
+pub use node_client::*;
```
Comment (Sumrachek) [inline ton_sdk/src/lib.rs]: Please, list types you import. ::* is now prohibited in tonlabs

Comment (AlexeyVavilin) [reply ton_sdk/src/lib.rs]: Done

File: ton_sdk/src/lib.rs (hunk 1 above)
Comment (AlexeyVavilin) [reply ton_sdk/src/lib.rs]: Done

File: ton_sdk/src/node_client.rs [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+            }
+        }
+
+        Ok(NodeClient {
+            client: GqlClient::new(&config.queries_server,&config.subscriptions_server)?,
+            timeouts: config.timeouts.unwrap_or_default()
+        })
+    }
+
+    pub fn get_timeouts(&self) -> &TimeoutsConfig {
```
[the remark this answers is not in the document]
Comment (AlexeyVavilin) [reply ton_sdk/src/node_client.rs]: Done

**Pull request merged**